source/patterns/@aws-solutions-constructs/core/lib/cloudfront-distribution-defaults.ts

```diff
@@ -42,11 +42,12 @@ export function DefaultCloudFrontWebDistributionForApiGatewayProps(apiEndPoint:
 
 export function DefaultCloudFrontWebDistributionForS3Props(sourceBucket: s3.IBucket, loggingBucket: s3.Bucket | undefined,
   setHttpSecurityHeaders: boolean,
+  originPath?: string | '/',
   cfFunction?: cloudfront.IFunction): cloudfront.DistributionProps {
 
   return {
     defaultBehavior: {
-      origin: new origins.S3Origin(sourceBucket),
+      origin: new origins.S3Origin(sourceBucket, { originPath }),
```

Comment:
So not providing an origin path and providing an origin path of / gives the exact same CloudFormation?

Reply:
The exact same CloudFormation as before? Yes, from my testing, that's what it looks like.

Reply:
Would you like me to add that test?

Reply:
No, it's fine as is.